packages/template-retail-react-app/app/ssr.js

```diff
@@ -92,6 +96,30 @@ const {handler} = runtime.createHandler(options, (app) => {
         res.send()
     })
 
+    app.post('/passwordless-login-callback', express.json(), async (req, res) => {
+        const base = req.protocol + '://' + req.get('host')
+        const {email_id, token} = req.body
+        const magicLink = `${base}/passwordless-login-landing?token=${token}`
+        const emailLinkResponse = await emailLink(
+            email_id,
+            process.env.MARKETING_CLOUD_PASSWORDLESS_LOGIN_TEMPLATE,
+            magicLink
+        )
+        res.send(emailLinkResponse)
+    })
+
+    app.post('/reset-password-callback', express.json(), async (req, res) => {
+        const base = req.protocol + '://' + req.get('host')
+        const {email_id, token} = req.body
+        const magicLink = `${base}/reset-password-landing?token=${token}`
+        const emailLinkResponse = await emailLink(
+            email_id,
+            process.env.MARKETING_CLOUD_RESET_PASSWORD_TEMPLATE,
+            magicLink
+        )
+        res.send(emailLinkResponse)
+    })
+
```

Comment:
These are both worthy to have a comment saying what they are on top of them. Also, if I squint they almost look like they have the same definition. Is there any room to reduce code duplication?